app/services/anonymized_intake_csv_service.rb

```diff
@@ -80,8 +80,39 @@ def decorated_intake(intake)
 
   private
 
+  def csv_headers
+    status_headers = EitcZendeskInstance::INTAKE_STATUS_LABELS.values.map { |label| "Intake Status - #{label}" } +
+                     EitcZendeskInstance::RETURN_STATUS_LABELS.values.map { |label| "Return Status - #{label}" } +
+                     EitcZendeskInstance::EIP_STATUS_LABELS.values.map { |label| "EIP Status - #{label}" }
+    CSV_HEADERS + status_headers
+  end
+
   def csv_row(intake)
-    CSV_FIELDS.map { |field| intake.send(field) }
+    row = CSV_FIELDS.map { |field| intake.send(field) }
+
+    # Add status transition times
+    intake_status_timestamps = {}
+    return_status_timestamps = {}
+    eip_status_timestamps = {}
+    intake.ticket_statuses.where(verified_change: true).order('created_at').each do |ticket_status|
+      unless intake_status_timestamps.key?(ticket_status.intake_status)
+        intake_status_timestamps[ticket_status.intake_status] = ticket_status.created_at
+      end
+
+      unless return_status_timestamps.key?(ticket_status.return_status)
+        return_status_timestamps[ticket_status.return_status] = ticket_status.created_at
+      end
+
+      unless eip_status_timestamps.key?(ticket_status.eip_status)
+        eip_status_timestamps[ticket_status.eip_status] = ticket_status.created_at
+      end
```

Comment:
i may be confused about something here - could there be multiple ticket statuses of a given type? if so, are you just recording the first one encountered? why?

Reply:
Good question. I'll push a follow-up commit to this PR that hopefully answers this. Happy to discuss.